MOKApp/app/src/main/java/mok/it/app/mokapp/recyclerview/ProjectsAdapter.kt

```diff
@@ -28,14 +28,16 @@ class ProjectsAdapter(
     }
 
     override fun onBindViewHolder(viewHolder: ViewHolder, position: Int) {
-        val model = dataSet[position]
+        val projectValuePair = dataSet[position]
         val ivImg: ImageView = viewHolder.itemView.findViewById(R.id.imageView)
         val tvBadgeName: TextView = viewHolder.itemView.findViewById(R.id.projectName)
-        loadImage(ivImg, model.icon)
-        tvBadgeName.text = model.name
+        val tvCollectedBadgeCount: TextView = viewHolder.itemView.findViewById((R.id.collectedBadgeCount))
+        loadImage(ivImg, projectValuePair.first.icon)
+        tvBadgeName.text = projectValuePair.first.name
+        tvCollectedBadgeCount.text = projectValuePair.second.toString()
 
         viewHolder.itemView.setOnClickListener {
-            listener.onProjectClicked(model.id)
+            listener.onProjectClicked(projectValuePair.first.id)
         }
     }
```

Comment:
A vízszintes recyclerviewban a projekteket megjelenítő kártyákról lelóg a mancsszám. Egyébként szerintem kicsit összezavaró az is, hogy saját mancsaimban ugyanúgy van jelölve a megszerzett mancsszám, mint ahogyan az AllProjects-ben a maximális mancsszám.